change/@fluentui-react-shared-contexts-33eb259a-36f1-46fd-91d6-e2e687089551.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add Tooltip context and TooltipManager interface",
```

Should this be the following?

```suggestion
  "comment": "Add TooltipContext and TooltipManager interface",
```